src/api/authed/intake/neighborhood-finder.js

```diff
@@ -0,0 +1,45 @@
+const {
```

Comment:
hmm i'm actually not sure if this belongs under `/authed`. it was suggested in #24 discussion but i don't think that's the right place anymore. please clarify if you know!

Reply:
Totally agree! Let's move it out of the authed endpoints